src/SensitiveFloats.jl

```diff
@@ -161,22 +161,15 @@ function combine_sfs!{ParamType <: ParamSet,
         sf_result::SensitiveFloat{ParamType, T1},
         v::T1, g_d::Vector{T2}, g_h::Matrix{T3};
         calculate_hessian::Bool=true)
-
-    # TODO: time consuming **************
-
-    # TODO: this line is allocating a lot of memory and I don't know why.
-    # Commenting this line out attributes the same allocation to the next line.
-    # Is memory being allocated lazily or misattributed?
-    @assert g_h[1, 2] == g_h[2, 1]
```

Comment:
I deleted the assertion since you don't usually want to test floating point numbers for exact equality.